apps/nextjs/next.config.ts

```diff
@@ -1,13 +1,8 @@
-import { fileURLToPath } from "url";
-import { createJiti } from "jiti";
+import type { NextConfig } from "next";
 
-// Import env files to validate at build time. Use jiti so we can load .ts files in here.
-await createJiti(fileURLToPath(import.meta.url)).import("./src/env");
+import "./src/env";
```

Comment:
Until next supports ESM in ts configs, I prefer to keep this .js

Reply:
It doesn't work? I tested and worked just fine. Is there any other reason?

Reply:
Try doing something ESM like accessing import.meta or similar.